assets/controllers/hello_controller.js

```diff
@@ -0,0 +1,16 @@
+import { Controller } from 'stimulus';
+
+/*
+ * This is an example Stimulus controller!
+ *
+ * Any element with a data-controller="hello" attribute will cause
+ * this controller to be executed. The name "hello" comes from the filename:
+ * hello_controller.js -> "hello"
+ *
+ * Delete this file or adapt it for your use!
+ */
+export default class extends Controller {
+    connect() {
+        this.element.textContent = 'Hello Stimulus! Edit me in assets/controllers/hello_controller.js';
```

Comment:
I guess this is was created by a recipe, but I don't think we need it for now

Reply:
Exactly, the new version of encore... I prefer to leave it. Do you see any problems?

Reply:
Well, it's just a piece of useless javascript. I don't see any reason to keep it

Reply:
You win! done! ;)